public/app/plugins/datasource/prometheus/querycache/QueryCache.ts

```diff
@@ -1,33 +1,36 @@
+import { toInteger } from 'lodash';
```

Comment:
what's wrong with `parseInt(value)`?

Reply:
I don't even think we need the parseInt anymore, that value we were using was already a number, should be fixed now.